src/backend/vulkan/src/conv.rs

```diff
@@ -208,86 +208,86 @@ pub fn map_mip_filter(filter: image::Filter) -> vk::SamplerMipmapMode {
 pub fn map_wrap(wrap: image::WrapMode) -> vk::SamplerAddressMode {
     use hal::image::WrapMode as Wm;
     match wrap {
-        Wm::Tile   => vk::SamplerAddressMode::Repeat,
-        Wm::Mirror => vk::SamplerAddressMode::MirroredRepeat,
-        Wm::Clamp  => vk::SamplerAddressMode::ClampToEdge,
-        Wm::Border => vk::SamplerAddressMode::ClampToBorder,
+        Wm::Tile   => vk::SamplerAddressMode::REPEAT,
+        Wm::Mirror => vk::SamplerAddressMode::MIRRORED_REPEAT,
+        Wm::Clamp  => vk::SamplerAddressMode::CLAMP_TO_EDGE,
+        Wm::Border => vk::SamplerAddressMode::CLAMP_TO_BORDER,
     }
 }
 
 pub fn map_border_color(col: image::PackedColor) -> Option<vk::BorderColor> {
     match col.0 {
-        0x00000000 => Some(vk::BorderColor::FloatTransparentBlack),
-        0xFF000000 => Some(vk::BorderColor::FloatOpaqueBlack),
-        0xFFFFFFFF => Some(vk::BorderColor::FloatOpaqueWhite),
+        0x00000000 => Some(vk::BorderColor::FLOAT_TRANSPARENT_BLACK),
+        0xFF000000 => Some(vk::BorderColor::FLOAT_OPAQUE_BLACK),
+        0xFFFFFFFF => Some(vk::BorderColor::FLOAT_OPAQUE_WHITE),
         _ => None
     }
 }
 
 pub fn map_topology(prim: Primitive) -> vk::PrimitiveTopology {
     match prim {
-        Primitive::PointList              => vk::PrimitiveTopology::PointList,
-        Primitive::LineList               => vk::PrimitiveTopology::LineList,
-        Primitive::LineListAdjacency      => vk::PrimitiveTopology::LineListWithAdjacency,
-        Primitive::LineStrip              => vk::PrimitiveTopology::LineStrip,
-        Primitive::LineStripAdjacency     => vk::PrimitiveTopology::LineStripWithAdjacency,
-        Primitive::TriangleList           => vk::PrimitiveTopology::TriangleList,
-        Primitive::TriangleListAdjacency  => vk::PrimitiveTopology::TriangleListWithAdjacency,
-        Primitive::TriangleStrip          => vk::PrimitiveTopology::TriangleStrip,
-        Primitive::TriangleStripAdjacency => vk::PrimitiveTopology::TriangleStripWithAdjacency,
-        Primitive::PatchList(_)           => vk::PrimitiveTopology::PatchList,
+        Primitive::PointList              => vk::PrimitiveTopology::POINT_LIST,
+        Primitive::LineList               => vk::PrimitiveTopology::LINE_LIST,
+        Primitive::LineListAdjacency      => vk::PrimitiveTopology::LINE_LIST_WITH_ADJACENCY,
+        Primitive::LineStrip              => vk::PrimitiveTopology::LINE_STRIP,
+        Primitive::LineStripAdjacency     => vk::PrimitiveTopology::LINE_LIST_WITH_ADJACENCY,
```

Comment:
this should be `LINE_STRIP_WITH_ADJACENCY`

Reply:
Fixed